evergreen.yml

```diff
@@ -190,12 +190,32 @@ functions:
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        local_file: mongo-tableau-connector/mongodb_jdbc-signed.taco
+        local_file: mongo-tableau-connector/mongodb_jdbc.taco
         remote_file: mongo-tableau-connector/artifacts/${version_id}/mongodb_jdbc-signed.taco
         bucket: mciuploads
         permissions: public-read
         content_type: application/octet-stream
 
+  "get signed connector":
```

Comment:
We have a `fetch signed connector` command already that you can use instead of adding this one.